src/common/i18n/he.json

```diff
@@ -32,6 +33,16 @@
     "export.dialog-info.link.label": "קישור:",
     "export.dialog-info.link.pattern": "https://packages/{packageName}.gpkg",
 
+    "export-table.dialog.title": "חבילות",
+    "export-table.link.text": "הורד",
+    "export-table.table-column-header.fileName.text": "שם הקובץ",
+    "export-table.table-column-header.sizeEst.text": "(Mb)גודל משואר",
+    "export-table.table-column-header.tilesEst.text": "מס' טיילים משואר",
+    "export-table.table-column-header.status.text": "סטטוס",
```

Comment:
spelling

Reply:
For my opinion "סטטוס" is good, you proposition?
Probably the better way is to give English version to Technical writer for Hebrew translation at the END